WebApiCoreSeed.WebApi/Filters/ValidateModelAttribute.cs

```diff
@@ -1,8 +1,10 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace WebApiCoreSeed.WebApi.Filters
 {
+    [ProducesResponseType(typeof(Dictionary<string, List<string>>), 400)]
```

Comment:
Nice! I did not know that Swagger searches metadata in the attributes.

Reply:
oops, that is a remanent of a test, ty.

Reply:
Oh, but does it work?

Reply:
no, it would be a nice, but i resolve the same issue with the other attribute that auto defines the output wherever you use the validationmodel attribute